src/main/java/com/beis/subsidy/award/transperancy/dbpublishingservice/service/AwardService.java

```diff
@@ -0,0 +1,246 @@
+package com.beis.subsidy.award.transperancy.dbpublishingservice.service;
+
+import java.math.BigDecimal;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.beis.subsidy.award.transperancy.dbpublishingservice.model.Award;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.model.Beneficiary;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.model.BulkUploadAwards;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.model.GrantingAuthority;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.model.SubsidyMeasure;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.repository.AwardRepository;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.repository.BeneficiaryRepository;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.repository.GrantingAuthorityRepository;
+import com.beis.subsidy.award.transperancy.dbpublishingservice.repository.SubsidyMeasureRepository;
+
+import lombok.extern.slf4j.Slf4j;
+@Slf4j
+@Service
+public class AwardService {
+
+	private static final String CONSTANT_SYSTEM = "SYSTEM";
```

Comment:
Create interface  ,declare methods and add implementation class add business logic.